readme.md

```diff
@@ -40,50 +41,56 @@ Hello, Node!
 
 * **Your Answer:**
 
----
+## "cat classmates.txt" for reading the content of the file* ##*"echo classmates.txt" for updating and in this case for appending new content to the file.\_ \_"rm classmates.txt" for deleting a file.
```

Comment:
To be clear, the `echo` command just prints out something to your console. It's the `>>` command that will add something to the end of a file. `touch` can also be used to create a file.